Insolvency.CalculationsEngine.Redundancy.BL/Services/Implementations/ArrearsOfPayCalculationsService.cs

```diff
@@ -71,7 +72,7 @@ public class ArrearsOfPayCalculationsService : IArrearsOfPayCalculationsService
 
                     //is this day a working day?
                     if (await date.IsEmploymentDay(data.ShiftPattern))
-                    {                        

```

Comment:
Please avoid reformatting files as part of functionality commits in future. It makes the diffs bigger than they need to be.